ice/routes/app.py

```diff
@@ -28,6 +30,20 @@
     )
 
 
+PING_RESPONSE = "ought-ice says pong"
+
+
+@app.get("/ping")
+async def ping():
+    return PlainTextResponse(PING_RESPONSE)
+
+
+@app.post("/stop")
+async def stop():
+    # Note that this doesn't work properly when the server runs with --reload
+    os.kill(os.getpid(), signal.SIGKILL)
```

Comment:
how about SIGTERM?

Reply:
sys.exit gets caught and responds with a 500.

SIGTERM works, but:

1. It's slower, so the stop command has to wait before checking if the server is still alive after sending the request.
2. I feel like I'm still supposed to follow up with SIGKILL after some time to make sure, in which case I might as well just SIGKILL from the beginning. I don't really know what SIGTERM gains.